csharp/test/Microsoft.ML.OnnxRuntime.Tests.NetCoreApp/InferenceTest.netcore.cs

```diff
@@ -476,22 +474,18 @@ private void TestPreTrainedModels(string opset, string modelName)
 
                 if (validModelFound)
                 {
-                    onnxModelFileName = Path.Combine(modelDir.FullName, onnxModelNames[0].Name);
+                    onnxModelFileName = Path.Combine(modelDirPath, onnxModelNames[0].Name);
                 }
                 else
                 {
                     var modelNamesList = string.Join(",", onnxModelNames.Select(x => x.ToString()));
-                    throw new Exception($"Opset {opset} Model {modelName}. Can't determine model file name. Found these :{modelNamesList}");
+                    throw new Exception($"Folder {modelDirPath} doesn't contain valid ONNX model file(s). Found these :{modelNamesList}");
                 }
 
                 using (var session = new InferenceSession(onnxModelFileName))
                 {
                     var inMeta = session.InputMetadata;
                     string testDataDirNamePattern = "test_data*";
-                    if (opset == "opset9" && modelName == "LSTM_Seq_lens_unpacked")
```

Comment:
does this special case of test data dir name mismatch still need to be handled?

Reply:
This opset is not used and tested anymore. For Nuget, it always uses the models in onnx submodule under onnxruntime.

Reply:
maybe we should check the test data directory for opset9/LSTM_Seq_lens_unpacked. looking at the test code below, if it doesn't match any directories in opset9/LSTM_Seq_lens_unpacked/test_data* it won't test anything.